src/lib.rs

```diff
@@ -64,8 +74,26 @@ pub fn drop_table(table: &str) -> io::Result<()> {
     Ok(())
 }
 
+#[allow(unused_must_use)]
```

Comment:
Here is the new diff: https://github.com/pindell-matt/fe_bucket/pull/7/commits/b6a1b369aa86020b74a750ceaad0abfff9fcd300

I still get the `unused or unknown feature, #[warn(unused_features)] on by default
src\lib.rs:1 #![feature(custom_derive, plugin, test)]` warning though